pkg/apis/openebs.io/device/v1alpha1/devicevolume.go

```diff
@@ -95,6 +96,10 @@ type VolumeError struct {
 // specific class of errors.
 type VolumeErrorCode string
 
+func (e *VolumeError) Error() string {
+	return e.Message
```

Comment:
nit: You may combine error message and code together something like `fmt.Sprintf("%s - %s", code, msg)`

Reply:
resolved